backend/compact-connect/lambdas/provider-data-v1/tests/function/test_handlers/test_provider_users.py

```diff
@@ -0,0 +1,63 @@
+import json
+
+from moto import mock_aws
+from tests.function import TstFunction
+from exceptions import CCInternalException
+
+TEST_COMPACT = "aslp"
+MOCK_SSN = "123-12-1234"
+@mock_aws
+class TestGetProvider(TstFunction):
+
+    def _create_test_provider(self):
+        from config import config
+        provider_id = config.data_client.get_or_create_provider_id(compact=TEST_COMPACT, ssn=MOCK_SSN)
+
+        return provider_id
+
+    def _when_testing_provider_user_event_with_custom_claims(self):
+        self._load_provider_data()
+        provider_id = self._create_test_provider()
+        with open('tests/resources/api-event.json', 'r') as f:
+            event = json.load(f)
+            event['requestContext']['authorizer']['claims']['custom:providerId'] = provider_id
+            event['requestContext']['authorizer']['claims']['custom:compact'] = TEST_COMPACT
+
+        return event
+
+    def test_get_provider_returns_provider_information(self):
+        from handlers.provider_users import get_provider_user_me
```

Comment:
Not something I've noticed either way in past tests, but any reason to not hoist this import to the top of the file? Or is this the convention we've been following for tests?

Reply:
It has to do with the way our imports pull in a global `config` module. When a handler file is imported, they in turn import a `config` module which then sets global properties used across the entire codebase. If you attempt to import the handlers at the top of the file, it will import the config file and set the global properties, which are then shared across all the other tests. This can cause certain tests to fail as a side effect. @jusdino established this config pattern to simplify accessing read-only variables that do not change in a production environment, but as a result of this pattern we are required to move the handler import statements inside of the tests.

Reply:
Got it. That makes sense!